modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -92,11 +93,11 @@
     /**
      * The constructor.
      *
-     * @param lockId Lock id.
+     * @param tableId Table id.
      * @param store  The storage.
      */
-    public PartitionListener(IgniteUuid lockId, VersionedRowStore store) {
-        this.lockId = lockId;
+    public PartitionListener(UUID tableId, VersionedRowStore store) {
+        this.lockId = new IgniteUuid(tableId, 0);
```

Comment:
Can UUID be used instead?
@ascherbakoff is there any reason to have IgniteUuid here?

Reply:
I decided not to change it for two reasons:
- I'm not as confident with all these locks
- this would lead to much more changes, it's out of scope in my opinion

Reply:
Does it make sense to create a ticket for this?

Reply:
I'm sure this part will be revisited while optimizing transactions